### PR TITLE
Improved hover effect for the buttons on the navbar

### LINKED ISSUE #404: [Style] Improve hover effect for the buttons on the navbar
### What's the style idea?

The current hover effects and animations for the nav-bar do not look complete or proper. I can improve the nav-bar drastically in terms of look and feel.

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [ ] I am a IWOC'24 contributor.
- [ ] I am a JWOC'24 contributor.
- [X] I am a SWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CODE_OF_CONDUCT.md)".

### PR DESCRIPTION
**Pull request by AftabMankapure**
# Fixes Issue🛠️ 

 
Closes #404 

# Description👨‍💻 

Added new hover effect 

# Type of change📄

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How this has been tested✅

# Checklist✅ 

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added demonstration in the form of GIF/video file
- [x] I am an Open Source Contributor

# Screenshots/GIF📷
Screenshot 2024-02-23 130534

### CODE REVIEW

**Comment by AftabMankapure** [comment]
@Rakesh9100  pls check this PR  it is new trading bouncing nav hover effect

this is my final PR for this issue because i try many of new hover effect but it affect navbar link navbar is not working because of hover effect 
if you like then merge otherwise you can assign to anyone

**Comment by Rakesh9100** [comment]
While hovering, the buttons should not move left @AftabMankapure

**Comment by AftabMankapure** [comment]
@Rakesh9100  pls  check it now

File: style.css [hunk 1]
```diff
@@ -337,10 +307,50 @@ nav li {
...
+    background-color: #3498db;
+    color: #df87ef;
+
+}
+
+.nav-menu .nav-item.active .nav-link {
+    color: red;
+}
+
+.nav-ite:before {
```
Comment (Rakesh9100) [inline style.css:322]: Is `nav-ite` class present anywhere? @AftabMankapure

**Comment by AftabMankapure** [comment]
@Rakesh9100  pls check it now

**Comment by Rakesh9100** [comment]
Merged 🎉 @AftabMankapure

**Pull request merged**